packages/components/addon/components/hds/app-footer/link.hbs

```diff
@@ -0,0 +1,23 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::AppFooter::Item>
+  {{! Q: Use Hds::Interactive for link or just Link::Inline? }}
+  <Hds::Link::Inline
+    class="hds-app-footer__link"
+    @color="secondary"
+    @route={{@route}}
+    @models={{hds-link-to-models @model @models}}
+    @query={{hds-link-to-query @query}}
+    @current-when={{@current-when}}
+    @replace={{@replace}}
+    @isRouteExternal={{@isRouteExternal}}
+    @href={{@href}}
+    @isHrefExternal={{@isHrefExternal}}
+    ...attributes
+  >
+    {{yield}}
+  </Hds::Link::Inline>
```

Comment:
This currently uses a different text color for the "default" links than the Secondary Link::Inline. Other styles for states are still not finalized. Also still need to handle theme colors for planned light & dark themes.

Reply:
Going to resolve this as part of clean up and as we agreed to do theming later on.